src/ray/core_worker/experimental_mutable_object_manager.cc

```diff
@@ -428,6 +442,18 @@ Status MutableObjectManager::SetErrorAll() {
   return ret;
 }
 
+std::unique_ptr<std::chrono::steady_clock::time_point>
+MutableObjectManager::ToTimeoutPoint(int64_t timeout_ms) {
+  if (timeout_ms == -1) {
+    return nullptr;
+  }
+  auto now = std::chrono::steady_clock::now();
+  auto timeout_duration = std::chrono::milliseconds(timeout_ms);
+  std::unique_ptr<std::chrono::steady_clock::time_point> timeout_point =
```

Comment:
According to the Google style guide, it's actually ok to use `auto` here instead of the type, since the right side of the assignment operator is basically a repeat of the type itself